pkg/server/unit_manager.go

```diff
@@ -233,14 +242,19 @@ func (um *UnitManager) StartUnit(podname, hostname, unitname, workingdir, netns
 	return nil
 }
 
-func (um *UnitManager) CaptureLogs(name string, unit *Unit) {
-	// XXX: Make number of log lines retained configurable.
-	lp := unit.LogPipe
-	um.logbuf.Set(name, logbuf.NewLogBuffer(logBuffSize))
+func (um *UnitManager) CaptureLogs(podName, unitName string, lp *LogPipe) {
+	namespace, name := util.SplitNamespaceAndName(podName)
+	cid := fmt.Sprintf("%d", time.Now().UnixNano())
+	logFileName := fmt.Sprintf(
+		"%s/%s_%s_%s-%s.log", ContainerLogDir, name, namespace, unitName, cid)
+	writer := containerlog.NewLogger(logFileName, 10, 1, 7, nil)
```

Comment:
Doesn't need any changes but just wanted to share a datapoint: When we changed our Ruby on Rails logging level to `debug` in production, a single webserver would create a 20-30GB log file every day.  That rate of logging would fill a 10MB file every 45 seconds.  Lets see how the 10MB setting goes.

Reply:
> Looks good. I assume the suggested logging pattern would be to run a log shipper sidecar that mounts the log directory (/var/log/containers/) as a HostPath volume?

We install the official google fluentd agent on our GCE images, running on the host, taking care of both container logs and system logs (including itzo logs).

Reply:
> Doesn't need any changes but just wanted to share a datapoint: When we changed our Ruby on Rails logging level to `debug` in production, a single webserver would create a 20-30GB log file every day. That rate of logging would fill a 10MB file every 45 seconds. Lets see how the 10MB setting goes.

That's an interesting data point. Let me change it to 100MB, which is more reasonable even for extremely busy and chatty applications, but still should be fine disk space-wise.